src/dataflow/src/render/reduce.rs

```diff
@@ -126,25 +124,29 @@ where
                         // If any error occurs we produce both the error as output,
```

Comment:
Seems like this comment is no longer true?

Reply:
That's true. Also one earlier on wasn't really correct either (said we "note whether errors got produced" but in fact we just return). I scratch both of the detailed comments, and I don't think they said anything surprising (essentially: "evaluate things and return an error if it happens").